index.html

```diff
@@ -0,0 +1,209 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css"/>
+    <title>Mustafa Portfolio</title>
+</head>
+<body> 
+    <section class="container">
+        <header class="header">
+            <a href="#"><h2>Mustafa Abbas</h2></a>
+            <nav class="humberger">
+            <img  src="./assets/img/humberger.png" alt="humberger"/>
+            <img  src="./assets/img/humberger.png" alt="humberger"/>
+            <img  src="./assets/img/humberger.png" alt="humberger"/>
+            </nav>
+            <ul class="main-nav">
+                <li><a href="#">Portoflio</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li>
+                <li ><a href="#"><img class="email-pic" src="./assets/img/Icon-Mail.png" alt="email"/></a></li >
+            </ul>
+        </header>
+        <div class="head-wraper">
+            <div class="title-wraper">
+                <h1>Mustafa Abbas</h1>
+                <h2>Software Developer</h2>
+            </div>
+        </div>
+        <div class="box"><p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,  don’t hestiate to contact me.</p></div>
+        <div class="line"></div>
+        <div class="skills">
+        <ul>
+            <li>Language</li>
+            <li>Javascript</li>
+            <li>Ruby</li>
+            <li>Html</li>
+            <li>Css</li>
+        </ul>
+        <ul>
+            <li>Frameworks</li>
+            <li>Ruby on Rails</li>
+            <li>Bootstrap</li>
+            <li>RSPec</li>
+            <li>CapyBara</li>
+        </ul>
+        <ul>
+            <li>Skills</li>
+            <li>Codekit</li>
+            <li>Github</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+        </ul>
+      </div>
+      <div class="line"></div>
+    </section>
+        <div class="footer-nav">
```

Comment:
```suggestion
        <div class="footer-nav">
```
better to include inside section tag here

Reply:
- Thanks for the polite words used here.
- [ ] I think you can add an explanation on why this change is required. For example, you can say 'semantically it is appropriate to wrap this section with a `section` tag'.